base/error.jl

```diff
@@ -77,6 +77,33 @@ function _reformat_bt(bt, bt2)
     ret
 end
 
+# Same as above, but inspects a single, stacked array to parse out interepreter frames
+function _reformat_bt(bt)
+    get_ip(frame::Ptr) = UInt(frame)
```

Comment:
Why not just split these out into e.g. `_frameip` and `_frameptr` outside of this function?

Reply:
I'm not sure what you mean?

Reply:
You have them as functions local to this function. If you move them outside, they can be compiled separately and added to the global method table and all that good stuff.

Reply:
They're not used anywhere else, so I hide them away in here so that they don't get added to the global method table.  They're just going to be inlined and optimized away anyway, so this seems the most sane option.

Reply:
Defining functions inside of functions, especially when they use dispatch, always feels like code smell to me, but I guess that's just me.

Reply:
"resolved" in https://github.com/JuliaLang/julia/pull/33467